resources/authgear/templates/ja/translation.json

```diff
@@ -0,0 +1,1041 @@
+  "settings-profile-edit-phone-number-placeholder": "電話番号がありません",
+  "settings-profile-edit-preferred-username-placeholder": "ユーザー名は利用できません",
+  "settings-profile-edit-upload-picture-label": "新しい写真をアップロード",
+  "settings-profile-edit-remove-picture-label": "写真を削除",
+  "settings-profile-edit-remove-confirmation-title": "プロフィール画像を削除",
+  "settings-profile-edit-remove-confirmation-desc": "プロフィール画像を削除してもよろしいですか？",
+  "settings-profile-edit-remove-confirmation-action-label": "削除",
+  "settings-profile-edit-remove-confirmation-cancel-label": "キャンセル",
+  "settings-page-anonymous-user": "ゲスト",
+  "standard-attribute-name": "フルネーム",
+  "standard-attribute-given-name": "名",
+  "standard-attribute-family-name": "姓",
+  "standard-attribute-middle-name": "ミドルネーム",
+  "standard-attribute-nickname": "ニックネーム",
+  "standard-attribute-picture": "プロフィール・ピクチャー",
+  "standard-attribute-profile": "プロフィール・アドレス",
+  "standard-attribute-website": "ウェブサイト URL",
+  "standard-attribute-email": "メインメールアドレス",
+  "standard-attribute-phone-number": "メイン電話番号",
+  "standard-attribute-preferred-username": "ユーザー名",
+  "standard-attribute-gender": "性別",
+  "standard-attribute-birthdate": "生年月日",
+  "standard-attribute-zoneinfo": "タイムゾーン",
+  "standard-attribute-locale": "優先言語",
+  "standard-attribute-address": "住所",
+  "standard-attribute-street-address": "住所",
+  "standard-attribute-locality": "市区",
+  "standard-attribute-postal-code": "郵便番号",
+  "standard-attribute-region": "州 / 省 / 府 / 地域",
+  "standard-attribute-country": "地域",
+  "custom-attribute-numeric-hint-minimum": "{minimum}以上の数字を入力してください",
+  "custom-attribute-numeric-hint-maximum": "{maximum}以下の数字を入力してください",
+  "custom-attribute-numeric-hint-minimum-maximum": "{minimum}から{maximum}の間の数値を入力してください",
+  "account-deletion-delete-account-link-label": "アカウントを削除する",
+  "account-deletion-title": "アカウントの削除",
+  "account-deletion-description": "アカウントを削除すると、すべてのデータが完全に削除され、復元できなくなります。続行すると、<span data-date=\"{rfc3339}\" data-date-type=absolute data-date-date-style=long data-date-time-style>{date, date, long}</span>にアカウントが削除されます。",
+  "account-deletion-confirmation-label": "<b>DELETE</b>と入力して確認してください",
+  "account-deletion-delete-account-button-label": "アカウントを削除",
+  "account-deletion-success-title": "アカウント削除のスケジュール",
+  "account-deletion-success-description": "アカウントが無効化され、データは<span data-date=\"{rfc3339}\" data-date-type=absolute data-date-date-style=long data-date-time-style>{date, date, long}</span>に削除されます",
+  "account-status-contact-support": "カスタマーサポートにお問い合わせください。",
+  "account-status-contact-support-with-link": "<a class=\"link\" target=\"_blank\" href={customerSupportLink}>カスタマーサポート</a>に連絡してください。",
+  "gender-female": "女性",
+  "gender-male": "男性",
+  "tutorial-signup-login": "これは、プロジェクトの<b>サインアップ&ログインページ</b>です。サインアップして、アプリの最初のユーザーになってみましょう!",
+  "tutorial-settings": "<b>アカウント設定ページ</b>の準備ができました。ユーザーはここでセキュリティとプロファイルを管理できます。完全な認証体験を提供するためにこのページをアプリに統合してください。",
+  "territory-AC": "アセンション島",
```

Comment:
We have a script scripts/npm/generate-territory-translation-json.js to obtain the localization from CLDR. Let's make new locales follow that.

Reply:
Seems cldr does not provide territory nor language names for es-ES and pt-BR, maybe I fallback to es and pt respectively?